src/Components/ProjectPage/ProjectCard.js

```diff
@@ -51,6 +51,7 @@ export default function ProjectCard(props) {
             display={display}
             frontstyle={classes.media}
             backstyle={classes.cardtext}
+            project={props.project}
```

Comment:
Line 43 seems to be not working properly.
props is 
![image](https://user-images.githubusercontent.com/39354695/87268956-744a8700-c4fe-11ea-86a5-7ed1ceeaf3d8.png)
therefore projectId is undefined